IdokladSdk.IntegrationTests/Tests/Clients/Attachment/AttachmentTests.cs

```diff
@@ -30,38 +32,75 @@ public void OneTimeSetUp()
         public void Upload_SuccessfullyUpdated()
         {
             // Arrange
-            var model = new AttachmentUploadModel
-            {
-                FileBytes = File.ReadAllBytes($"{TestContext.CurrentContext.TestDirectory}/{AttachmentPath}"),
-                FileName = FileName,
-                DocumentType = AttachmentDocumentType.IssuedInvoice,
-                DocumentId = DocumentId
-            };
+            var model1 = GetAttachmentUploadModel(1);
+            var model2 = GetAttachmentUploadModel(2);
+            var model3 = GetAttachmentUploadModel(3);
 
             // Act
-            var data = _attachmentClient.Upload(model).AssertResult();
+            var data1 = _attachmentClient.Upload(model1).AssertResult();
+            var data2 = _attachmentClient.Upload(model2).AssertResult();
+            var data3 = _attachmentClient.Upload(model3).AssertResult();
 
             // Assert
-            Assert.IsTrue(data);
+            Assert.IsTrue(data1);
+            Assert.IsTrue(data2);
+            Assert.IsTrue(data3);
         }
 
         [Test]
         [Order(2)]
-        public void Get_SuccessfullyGetAttachment()
+        public void Get_SuccessfullyGetAllAttachments()
         {
             // Act
             var data = _attachmentClient.Get(DocumentId, AttachmentDocumentType.IssuedInvoice).AssertResult();
 
             // Assert
-            var first = data.FirstOrDefault();
-            Assert.NotNull(first);
-            Assert.NotNull(first.FileBytes);
-            Assert.AreEqual(FileName, first.FileName);
+            Assert.Multiple(() =>
+            {
+                data.ForEach((attachment) =>
+                {
+                    Assert.That(attachment.Id, Is.Not.EqualTo(0));
+                    Assert.NotNull(attachment.FileBytes);
+                    Assert.NotNull(attachment.FileName);
+                });
+            });
+
+            var attachment = data.FirstOrDefault();
+            Assert.NotNull(attachment);
+
+            _attachmentId = attachment.Id;
+            _attachmentName = attachment.FileName;
+
+            Assert.That(_attachmentName, Is.Not.Null.Or.Empty);
```

Comment:
Tohle přece testuješ v tom Assert.Multiple, ne? Já bych asi na začátku assertoval, že data.count je větší než 0 a pak to můžeš vynechat, protože se to otestuje v tom Assert.Multiple.